src/test/java/Pages/GooglePage.java

```diff
@@ -0,0 +1,81 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class GooglePage {
+
+    private static final By REJECT_ALL = By.id("W0wltc");
+
+    private static final By SEARCH_FIELD = By.id("APjFqb");
+    private static final By SEARCH_RESULTS = By.xpath("//a/h3");
+    private static final By FEELING_LUCKY = By.id("gbqfbb");
+    //redirected to url that contains search term
+    private WebDriver driver;
+
+
+    public GooglePage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+
+    public void waitForElementToBeClickable(WebElement element) {
```

Comment:
Here you could return a `WebElement` rather than `void`. Then return your line 34 code and whenever you call this method then, you will be able to call the click/sendKeys methods on that so it will save you a line of code every where you use it.